packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -879,12 +907,36 @@ export class BrowserTool extends BaseBrowserTool {
 
   private cleanupScreenshots(): void {
```

Comment:
I don't think this is needed anymore, since no more screenshots are created within `screenshots` folder. Instead, we should clean up the legacy folder via a new function `purgeLegacyScreenshots`, similar to `purgeLegacyCache`.